core/src/test/java/org/elasticsearch/index/query/SpanNearQueryBuilderTest.java

```diff
@@ -63,7 +63,11 @@ public void testValidate() {
         int clauses = randomIntBetween(1, 10);
         for (int i = 0; i < clauses; i++) {
             if (randomBoolean()) {
-                queryBuilder.clause(new SpanTermQueryBuilder("", "test"));
+                if (randomBoolean()) {
+                    queryBuilder.clause(new SpanTermQueryBuilder("", "test"));
+                } else {
+                    queryBuilder.clause(null);
```

Comment:
yes I get that, my code would still go in the else block. I mean that setting null would be the same as not setting anything?